content/en/platform/corda/5.0/developing-applications/api/api-crypto-extensions.md

```diff
@@ -51,28 +54,154 @@ class TripleSha256Digest : DigestAlgorithm {
         return messageDigest.digest().sha256Bytes().sha256Bytes()
     }
 }
+
+private fun ByteArray.sha256Bytes(): ByteArray =
+    MessageDigest.getInstance(DigestAlgorithmName.SHA2_256.name).digest(this)
 ```
+{{% /tab %}}
+{{% tab name="Java" %}}
```

Comment:
Is there going to be another tab (Kotlin) ?
If not - then is there really a reason to use `{{% tabs %}}`

Reply:
> Is there going to be another tab (Kotlin) ? If not - then is there really a reason to use `{{% tabs %}}`

Not sure I understand.. there is a Kotlin tab defined above the Java one?

Reply:
Preview here: https://nadine.preview.docs.r3.com/en/platform/corda/5.0/developing-applications/api/api-crypto-extensions.html#triplesha256digestkt